cfme/tests/containers/test_relationships_tables.py

```diff
@@ -21,10 +19,10 @@
 pytest_generate_tests = testgen.generate(
     testgen.container_providers, scope="function")
 
+# CMP-9929 # CMP-9930 # CMP-9892 # CMP-9965 # CMP=9962 # CMP-9983
```

Comment:
please split it - the relevant id should be above the relevant test
this is not useful